.env

```diff
@@ -0,0 +1,30 @@
+STAGE_KEYPAIR_NAME="rhel_stage_id_rsa"
+STAGE_AMI="ami-d2414e38"
+STAGE_INSTANCE_TYPE="t2.micro"
+STAGE_REGION="eu-west-1"
+STAGE_SUBNET_ID="subnet-2c67fe64"
+STAGE_SECURITY_GROUPS="sg-7b78fe07"
+STAGE_SSH_PRIVATE_KEY_PATH="~/.ssh/rhel_stage_id_rsa.pem"
+
+PROD_KEYPAIR_NAME="rhel_aws_id_rsa"
+PROD_AMI="ami-d2414e38"
+PROD_INSTANCE_TYPE="t2.micro"
+PROD_ELASTIC_IP="52.209.230.146"
+PROD_REGION="eu-west-1"
+PROD_SUBNET_ID="subnet-547fcd1c"
+PROD_SECURITY_GROUPS="sg-078605556141c7c6c"
+PROD_SSH_PRIVATE_KEY_PATH="~/.ssh/rhel_aws_id_rsa.pem"
+
+BASE_FOLDER=/vagrant
+NAMESPACE=devopsloft
+DOCKERHUB_APPLICATION=devopsloft
```

Comment:
using DOCKERHUB in variable name is not recommended. Today it is DockerHub, tomorrow it is another docker registry.

Reply:
done